js/client/modules/@arangodb/testutils/testrunner.js

```diff
@@ -380,9 +381,7 @@ class testRunner {
     if (checkCollections) {
       this.cleanupChecks.push(graphsTest);
     }
-    this.cleanupChecks.push();
     this.instanceManager;
```

Comment:
What is the purpose of this line? It was there before already, but what does it do?